File: 1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.js
/**
 * @param {number} m
 * @param {number} n
 * @param {number[][]} indices
 * @return {number}
 */
var oddCells = function(m, n, indices) {
    const grid = Array(m).fill(0).map(() => Array(n).fill(0))
    for (const [r, c] of indices) {
        for (let col=0; col < n; col++) {
            grid[r][col]++
        }
        for (let row=0; row < m; row++) {
            grid[row][c]++
        }
    }
    let res = 0
    for (let r=0; r < m; r++) {
        for (let c=0; c < n; c++) {
            if (grid[r][c] % 2 !== 0) res++
        }
    }
    return res
};

File: 1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.py
class Solution:
    def oddCells(self, m: int, n: int, indices: List[List[int]]) -> int:
        grid = [[0] * n for _ in range(0, m)]
        for r, c in indices:
            for row in range(0, m):
                grid[row][c] += 1
            for col in range(0, n):
                grid[r][col] += 1
        return sum([1 if cell % 2 != 0 else 0 for row in grid for cell in row])
        return sum([1 for row in grid for cell in row if cell % 2 != 0])